Lightbox.js

```diff
@@ -34,11 +34,13 @@ var Lightbox = React.createClass({
       friction:      PropTypes.number,
     }),
     swipeToDismiss:  PropTypes.bool,
+    transform:       PropTypes.bool,
```

Comment:
This also includes the ability to move the image along the x/y axis and double tap functionality. In general, the property determines if the view transformer will be used in fullscreen or not